tests/e2e/specs/shopper/checkout-different-addresses.test.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	uiUnblocked,
+} from '@woocommerce/e2e-utils';
+
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getNormalPagePermalink,
+	shopper,
+	visitPostOfType,
+	preventCompatibilityNotice,
+	reactivateCompatibilityNotice,
+} from '../../../utils';
+
+const config = require( 'config' );
+const block = { name: 'Checkout' };
+const simpleProductName = '128GB USB Stick';
+const shippingDetails = config.get( 'addresses.customer.shipping' );
+const billingDetails = config.get( 'addresses.customer.billing' );
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Checkout → Can have different shipping and billing addresses', () => {
+	let productPermalink;
+
+	beforeAll( async () => {
+		// prevent CartCheckoutCompatibilityNotice from appearing
+		await preventCompatibilityNotice();
+		await merchant.login();
+
+		// Activate all address fields.
+		await visitBlockPage( 'Checkout Block' );
+		await openDocumentSettingsSidebar();
+		await selectBlockByName(
+			'woocommerce/checkout-shipping-address-block'
+		);
+		await expect( page ).toClick( 'label', { text: 'Company' } );
+		await expect( page ).toClick( 'label', {
+			text: 'Apartment, suite, etc.',
+		} );
+		await expect( page ).toClick( 'label', { text: 'Phone' } );
+		await saveOrPublish();
+
+		// Get product page permalink.
+		await visitPostOfType( simpleProductName, 'product' );
```

Comment:
WC has this [utility](https://www.npmjs.com/package/@woocommerce/e2e-utils#shopper-shopper) `goToProduct`, it's more straight forward and maybe faster to use that instead